lib/elixir/lib/module.ex

```diff
@@ -1201,6 +1201,6 @@ defmodule Module do
   defp assert_not_compiled!(fun, module) do
     open?(module) ||
       raise ArgumentError,
-        "could not call #{fun} on module #{inspect module} because it was already compiled"
+        "could not call #{fun} in module #{inspect module} because it was already compiled"
```

Comment:
I'm not sure about this particular change; here we're saying that we cannot call `fun` on `module` as in we cannot call `fun(module, ...)`. For example when this error is raised for `get_attribute`, the problem is that we can't call `get_attribute(module, ...)` but `get_attribute` is not "in" `module`. Did I make sense? :)